Makefile

```diff
@@ -0,0 +1,7 @@
+.PHONY: test test-rpc
+
+test:
+	@go test ./cmd/solana_exporter -v -short
+
+test-rpc:
+	@go test ./pkg/rpc -v
```

Comment:
Most go devs will just run:

```shell
go test ./...
```

We should try to come up with a better approach for this. I'm not super comfortable with this approach as I feel it is shoehorning something into the unit test framework that is not a unit test.